tensorbay/client/struct.py

```diff
@@ -25,6 +25,101 @@
 ROOT_COMMIT_ID = "00000000000000000000000000000000"
 
 
+class Team(AttrsMixin, ReprMixin):
+    """This class defines the basic concept of a TensorBay team.
+
+    Arguments:
+        name: The name of the team.
+        email: The email of the team.
+        description: The description of the team.
+
+    """
+
+    _T = TypeVar("_T", bound="Team")
+
+    _repr_attrs: Tuple[str, ...] = ("email",)
+
+    name: str = attr()
+    email: Optional[str] = attr(default=None)
+    description: str = attr(default="")
+
+    def __init__(self, name: str, *, email: Optional[str] = None, description: str) -> None:
+        self.name = name
+        self.email = email
+        self.description = description
+
+    def _repr_head(self) -> str:
+        return f'{self.__class__.__name__}("{self.name}")'
+
+    @classmethod
+    def loads(cls: Type[_T], contents: Dict[str, Any]) -> _T:
+        """Loads a :class:`Team` instance from the given contents.
+
+        Arguments:
+            contents: A dict containing all the information of the commit::
+
+                    {
+                        "name": <str>
+                        "email": <str>
```

Comment:
optional str?

Reply:
The attr won't be dumped when it is of its default value.